include/cudaq/Optimizer/Transforms/Passes.h

```diff
@@ -54,4 +52,18 @@ std::unique_ptr<mlir::Pass> createUnwindLoweringPass();
 #define GEN_PASS_REGISTRATION
 #include "cudaq/Optimizer/Transforms/Passes.h.inc"
 
+/// Helper to run the memory to register pass on classical values. Does not
+/// convert the quantum code to register (wire) form.
+inline std::unique_ptr<mlir::Pass> createClassicalMemToReg() {
+  MemToRegOptions m2rOpt = {/*classical=*/true, /*quantum=*/false};
+  return createMemToReg(m2rOpt);
+}
+
+/// Helper to run the memory to register pass on quantum wires. Does not convert
+/// classical code to register form.
+inline std::unique_ptr<mlir::Pass> createQuantumMemToReg() {
```

Comment:
Do we need another function for the case of classical=true and quantum=true?

Reply:
I don't think so. That is the default. For now anyway.